Alert/doc/spec.md

```diff
@@ -0,0 +1,155 @@
+# Alert data model
+
+## Description
+
+This entity models an alert and could be used to send alerts related to traffic
+jam, accidents, weather conditions, high level of pollutants and so on.
+The purpose of the model is to support the generation of notifications for
+a user or trigger other actions, based on such alerts.
+
+An alert is generated by a specific situation. The main features of an alert is
+that it is not predictable and it is not a recurrent data. That means that an
+alert could be an accident or a high level of pollutants measure, additionally
+it could be the fall down of a patient or a car driving in the opposite
+direction.
+
+Some examples of context data are: type of alert (traffic, weather, security,
+and pollution, etc.), severity, location and so on.
+
+## Data model
+
+A JSON Schema corresponding to this data model can be
+found [here](../schema.json).
+
++ `id` : Entity's unique identifier.
+
++ `type` : Entity type. It must be equal to `Alert`.
+
++ `category` : Define the category of alert (Traffic jam, accidents, weather
+  conditions, high level of pollutants)
+  + Attribute type: [Text](https://schema.org/Text)
+  + Allowed values:
+    + (`traffic`, `weather`, `environment`, `health`, `security` )
+  + Mandatory
+
++ `subCategory` : Describe the sub category of alert.
+  + Attribute type: [Text](https://schema.org/Text)
+  + Allowed values:
+    + (`trafficJam`, `carAccident`, `carWrongDirection`, `carStopped`,
+      `pothole`, `roadClosed`, `roadWorks`, `hazardOnRoad`, `injuredBiker`)
+      (for `traffic` category)
+    + (`rainfall`, `highTemperature`, `lowTemperature`, `heatWave`, `ice`,
+      `snow`, `wind`, `fog`, `flood`, `tsunami`, `tornado`, `tropicalCyclone`,
+      `hurricane` ) (for `weather` category)
+    + (`airPollution`, `waterPollution`, `pollenConcentration`)
+      (for `environment` category)
+    + (`asthmaAttack`, `bumpedPatient`, `fallenPatient`, `heartAttack` )
+      (for `health` category)
+    + (`suspiciousAction`, `robbery`, `assault` ) (for `security` category)
+  + Optional
+
++ `location` : Location of alert represented by a GeoJSON geometry.
+  + Attribute type: `geo:json`.
+  + Normative References: [rfc7946](https://tools.ietf.org/html/rfc7946)
+  + Mandatory if `address` is not present
+
++ `address` : Civic address of `Alert`
+  + Attribute type: [Address](https://schema.org/address)
+  + Mandatory if `location` is not present.
+
++ `dateTimeObserved` : The date and time the observation of the alert
+  in ISO8601 UTC format.
+  + Attribute type: [DateTime](https://schema.org/DateTime).
+  + Normative References: [ISO8601](https://www.iso.org/standard/40874.html)
+  + Mandatory
+
++ `dateTimeValidity` : The date and time of validity of the alert in ISO8601
```

Comment:
I don't agree with this attribute

We need `validFrom` and `validTo`

Reply:
removed the structured attribute and replaced with two separate flat attributes